mne/time_frequency/tfr.py

```diff
@@ -2022,21 +2044,9 @@ def __repr__(self):  # noqa: D105
 
     def __abs__(self):
         """Take the absolute value."""
-        return EpochsTFR(info=self.info.copy(), data=np.abs(self.data),
-                         times=self.times.copy(), freqs=self.freqs.copy(),
-                         method=self.method, comment=self.comment)
-
-    def copy(self):
-        """Give a copy of the EpochsTFR.
-
-        Returns
-        -------
-        tfr : instance of EpochsTFR
-            The copy.
-        """
-        return EpochsTFR(info=self.info.copy(), data=self.data.copy(),
-                         times=self.times.copy(), freqs=self.freqs.copy(),
-                         method=self.method, comment=self.comment)
+        epochs = self.copy()
```

Comment:
where is the copy() method implemented now?

Reply:
In the class _BaseTFR, mne/time_frequency/tfr.py:910 (line 899, in master).